test/cocotb/osdtestlib/modtestclasses.py

```diff
@@ -0,0 +1,637 @@
+import cocotb
+from cocotb.triggers import Timer
+from cocotb.clock import Clock
+from cocotb.result import TestFailure
+from cocotb.triggers import RisingEdge
+
+class NoC_Driver:
+    """
+    Class responsible for delivering individual words to a debug module and 
+    processing their reaction
+    """
+
+    @cocotb.coroutine
+    def build_flit(self, dut, data, last):
+        """function which transmits one flit between debug modules
```

Comment:
The method name says a flit is built, the comment says it's transmitted. What's correct?

Also, there's no need to include the "function which" part in the documentation. Simply write "Transmit one flit between modules".